tests/test_elastic_search_visitor.py

```diff
@@ -3429,3 +3429,32 @@ def test_range_date_queries_are_nested():
 
     generated_es_query = _parse_query(query_str)
     assert generated_es_query == expected_es_query
+
+
+def test_date_updated_keyword_is_handled_with_range_query():
+    query_str = 'du 2009->2019'
+    expected_es_query = {
+       "range": {
+          "_updated": {
+             "gte": "2009",
+             "lte": "2019"
+          }
+       }
+    }
+
+    generated_es_query = _parse_query(query_str)
+    assert generated_es_query == expected_es_query
+
+
+def test_date_added_keyword_is_handled_with_range_query():
+    query_str = 'da 2009'
+    expected_es_query = {
+       "range": {
+          "_created": {
+             "eq": "2009"
```

Comment:
shouldn't this be something like
```suggestion
             "gte": "2009",
             "lt": "2010"
```